app/Database/seeds/settings.yml

```diff
@@ -235,6 +235,27 @@
   options: ''
   type: number
   description: 'How much the load factor can vary per-flight'
+- key: flights.different_cargo_load_factor
+  name: 'Different Cargo Load Factor'
+  group: flights
+  value: false
+  options: ''
+  type: boolean
+  description: 'When enabled, v7 will use a different load factor for cargo and passengers fares'
+- key: flights.default_cargo_load_factor
```

Comment:
Why we are naming this as `default` , shouldn't be it just `flights.cargo_load_factor` like the variance ?

Reply:
there is this default for the passenger load factor, I wanted to keep the same structure

Reply:
Understood :)